mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/websocket/MqttWebSocketSubscriber.java

```diff
@@ -31,7 +31,8 @@ public void subscribe(MqttMessageListener listener) {
             return;
         }
 
-        final String token = tokenSupplier.get();
+        final String token = tokenService.get().getTokenString();
```

Comment:
After changing the name this doesn't look right, `get` what? At least change the name to `getToken`

Reply:
Maybe this relates to TokenService not being quite right (although better than TokenProvider). Maybe TokenContext is a better name as it hold relevant Token state (rather than providing a fuller toekn 'service')

Reply:
Sorry I thought it held the token value, but just looked again and it doesn't. So maybe not TokenContext. Maybe TokenServce is okay?

Reply:
Sorry, in 2 minds - TokenThing holds a token so maybe Context is better than Service.

Thoughts anyone?

Reply:
I think Service fits better given it handles unsubscribing as well. FYI TokenAPI handles unsubscribe action - so nothing to do with me. This class is mere reflect of what the Core API provides

Reply:
Done.

Reply:
BTW from the caller getToken().getTokenString() (the last part) does not look good - but I am not going to go and change the TokenAPI as well for now :)